### ISSUE TITLE
Import from Coding Rooms (addition to #206)

### ISSUE THREAD

**Issue by bishopmary** [JuiceMind/JuiceMind#217]
Import now works on gloomis69 files (from issue #206)

I have one that imports (CSA), but with issues I need to look at.

Two now progress past the 50% point ("Converting from Coding Rooms to Juiceminds"), but then error out -- one at over 90% done and the other at almost 60% (both still during "convert" phase). 

They are both too big to upload :(

**Comment by bishopmary** [comment]
Update #2:  I duplicated the same Coding Rooms course and kept only the last half -- to see if issue is the size or something in the file ... This one also bombed at over 90% complete (during "conversion" phase) -- so something in my course/zip?

[cr-course-2024-spring-focs--pyonlytst.zip](https://github.com/user-attachments/files/16088924/cr-course-2024-spring-focs--pyonlytst.zip)

**Comment by juicemind-dev** [comment]
Hi @bishopmary we tried importing your zip file and it works on our side now after some updates we made throughout the day. Can you give it another try?

**Comment by bishopmary** [comment]
Will do!!  Thanks!
   
   
   
    On 07/04/2024 12:40 AM EDT juicemind-dev ***@***.***> wrote:
    
   
     
    
   
     
    
     
   Hi @bishopmary we tried importing your zip file and it works on our side now after some updates we made throughout the day. Can you give it another try?

**Comment by juicemind-dev** [comment]
Hi @bishopmary we were wondering if you ever got the chance to take a look at the import again.

Thanks!

**Comment by bishopmary** [comment]
I did - thanks!  working on export/import of all courses now
   
  
    
   
  
   sorry I did not let you know --  I looked for the entry to reply but couldn't find it --  i guess I could've emailed lol
   
  
    
   
  
   thanks for all your hard work --  looks like a lot of good stuff happening!
   
   
   
    On 07/27/2024 7:50 AM EDT juicemind-dev ***@***.***> wrote:
    
   
     
    
   
     
    
     
   Hi @bishopmary we were wondering if you ever got the chance to take a look at the import again. 
   Thanks!
